openrtb_ext/bidders.go

```diff
@@ -15,7 +15,7 @@ import (
 // BidderName refers to a core bidder id or an alias id.
 type BidderName string
 
-var aliasBidderNames []BidderName
+var aliasBidderToParentBidder map[BidderName]BidderName = map[BidderName]BidderName{}
```

Comment:
Nitpick: `aliasBidderToParent` is likely good enough. The extra "bidder" seems a little redundant.

Reply:
https://github.com/prebid/prebid-server/pull/3048/commits/8e4cd4c9252d8b836b76b0833cb201c24d58aa3a